tt_metal/hw/inc/grayskull/core_config.h

```diff
@@ -8,3 +8,13 @@ enum ProgrammableCoreType {
     TENSIX     = 0,
     COUNT      = 3, // for now, easier to keep structures shared across arches' the same size
 };
+
+enum class AddressableCoreType : unsigned char {
+    TENSIX    = 0,
+    ETH       = 1,
```

Comment:
do we need eth for grayskull core conifg? or we want this to be symmetrical across all archs?

Reply:
Good catch, removed this

Reply:
Ah if we remove this we need to #ifdef out all uses on host size (and device side but it's simple there), going to leave it in for now

Reply:
the host side code should access this through the HAL, but that is future work...

Reply:
Added a TODO here